modules/migration-compatibility-guidelines.adoc

```diff
@@ -8,18 +8,45 @@
 [id="migration-compatibility-guidelines_{context}"]
 = Compatibility guidelines
 
+// Do we have a "What's New" section in the Release Notes? I'm wondering what, given the complicated support model we discuss here, is the motivation to upgrade from MTC 1.5/1.6 to MTC 1.7? Or is the usage model only: Install once, migrate, and that's it, you don't need MTC after that, so no need to upgrade...?
+
 You must install the {mtc-full} ({mtc-short}) Operator that is compatible with your {product-title} version.
 
-You cannot install {mtc-short} {mtc-version} on {product-title} 4.5, or earlier versions, because the custom resource definition API versions are incompatible.
+.Restrictions
+
+{product-title} 4.5 and earlier are referred to as _legacy platforms_. {product-title} 4.6 and later are referred to as _modern platforms_.
+
+* Legacy platforms are no longer supported.
+* You cannot install {mtc-short} 1.6 or {mtc-version} on legacy platforms, because the custom resource definition API versions are incompatible.
+//* You cannot install {mtc-short} 1.6 on {product-title} 4.5 or earlier versions, because the custom resource definition API versions are incompatible.
+* {mtc-short} 1.6 and later versions, including {mtc-version}, do not support the `MigrationController` custom resource and the {mtc-short} web console on legacy source clusters, which must run {mtc-short} {mtc-legacy-version-z}. So you must install {mtc-short} 1.6 or later, including {mtc-version}, on the destination cluster.
++
+[NOTE]
+====
+With {mtc-short} {mtc-version} or later, when migrating from a source cluster on a legacy platform to a destination cluster on a modern platform, use the `crane tunnel-api` command. See xxx.
+====
+// You can migrate workloads from a source cluster with {mtc-short} {mtc-legacy-version-z} to a target cluster with {mtc-short} {mtc-version} as long as the `MigrationController` custom resource and the {mtc-short} web console are running on the destination cluster.
```

Comment:
The `MigrationController` resource is simply a CR reconciled by the operator to install MTC components. It will always be present regardless of whether the cluster is a source or a target. This also isn't really an accurate statement. The controller does not necessarily have to be a destination cluster.

https://github.com/konveyor/mig-operator/blob/master/docs/usage/AlternativeCAMTopologies.md